scripts/build_and_push_images.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 #
 # Builds and pushes operator, bundle and index images for a given version of an operator
 #
-# NOTE: Requires opm and operator-sdk to be installed!
+# NOTE: Requires yq, make, podman, opm and operator-sdk to be installed!
```

Comment:
we don't require 'yq' again do we (I don't have it!)

Reply:
Ack